source/configuration/modules/omfile.rst

```diff
@@ -1,434 +1,441 @@
-omfile: File Output Module
-==========================
+omfile: Text File Output Module
+###############################
 
-===========================  ===========================================================================
-**Module Name:**             **omfile**
-**Author:**                  `Rainer Gerhards <http://www.gerhards.net/rainer>`_ <rgerhards@adiscon.com>
-===========================  ===========================================================================
+================  ===========================================================================
+**Module Name:**  **omfile**
+**Author:**       `Rainer Gerhards <http://www.gerhards.net/rainer>`_ <rgerhards@adiscon.com>
+================  ===========================================================================
 
-The omfile plug-in provides the core functionality of writing messages
-to files residing inside the local file system (which may actually be
-remote if methods like NFS are used). Both files named with static names
-as well files with names based on message content are supported by this
-module.
+The omfile plugin provides the needed functionality to write messages to a local file.
 
 Configuration Parameters
-------------------------
-Omfile is a built-in module that does not need to be loaded. In order to
-specify module parameters, use
+************************
 
-::
-
-   module(load="builtin:omfile" ...parameters...)
-
-Note that legacy directives **do not** affect new-style RainerScript configuration
-objects. See :doc:`basic configuration structure doc <../basic_structure>` to
-learn about different configuration languages in use by rsyslog.
+Omfile is a built-in module that does not need to be loaded for default settings.
+In order to specify module parameters, it must be loaded using:
 
-General Notes
-^^^^^^^^^^^^^
+    module(load="builtin:omfile" [parameters])
 
 As can be seen in the parameters below, owner and groups can be set either by
```

Comment:
Agree. I'm just reviewing not remaking docs while I'm learning to use each features. Probably that could be a future improvement.

Anyway, I'm open for suggestions on how to do it ;)